modules/dnn/src/op_webnn.cpp

```diff
@@ -0,0 +1,121 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018-2019, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include <fstream>
+#include "op_webnn.hpp"
+
+#include <opencv2/core/utils/configuration.private.hpp>
+#include <opencv2/core/utils/logger.hpp>
+
+#include "opencv2/core/utils/filesystem.hpp"
+#include "opencv2/core/utils/filesystem.private.hpp"
+
+#include <opencv2/dnn/shape_utils.hpp>
+
+namespace cv { namespace dnn {
+
+#ifdef HAVE_WEBNN
+
+// WebNNGraph
+
+WebNNGraph::WebNNGraph()
+{
+
+}
+
+bool WebNNGraph::isInitialized()
+{
+
+}
+
+void WebNNGraph::init(Target targetId)
+{
+
+}
+
+void WebNNGraph::forward(const std::vector<Ptr<BackendWrapper> >& outBlobsWrappers)
+{
+
+}
+
+void WebNNGraph::createGraph(Target targetId)
+{
+
+}
+
+// WebNNBackendNode
+
+WebNNBackendNode::WebNNBackendNode(const std::vector<Ptr<BackendNode> >& nodes,
+                                         Ptr<Layer>& cvLayer_, std::vector<Mat*>& inputs,
+                                         std::vector<Mat>& outputs, std::vector<Mat>& internals)
+    : BackendNode(DNN_BACKEND_WEBNN), cvLayer(cvLayer_)
+{
+    std::ostringstream oss;
+    oss << (size_t)cvLayer.get();
+
+    std::map<std::string, InferenceEngine::Parameter> params = {
```

Comment:
what's the usage of this variable?

Reply:
This is the implementation of Ngraph. I will replace it with Webnn implementation. All the relative codes have been deleted.